Cargo.toml

```diff
@@ -451,6 +451,11 @@ crossbeam-epoch = { git = "https://github.com/solana-labs/crossbeam", rev = "fd2
 # and we end up with two versions of `solana-program` and `solana-zk-token-sdk` and all of their
 # dependencies in our build tree.
 #
+# If you are developing downstream using non-crates-io solana-program (local or
+# forked repo, or from github rev, eg), duplicate the following patch statements
+# in your Cargo.toml. If you still hit duplicate-type errors with the patch
+# statements in place, refresh your Cargo.lock file.
```

Comment:
I'm worried that the term "refresh" might confuse some people.

What do you think about spelling it out completely and recommending that people run `cargo update -p solana-program` and / or `cargo update -p solana-zk-token-sdk`?

Reply:
Oh great idea, thank you!